ts/features/common/store/reducers/index.ts

```diff
@@ -98,7 +103,8 @@ const rootReducer = combineReducers<FeaturesState, Action>({
   itWallet: itWalletReducer,
   profileSettings: profileSettingsReducerPersistor,
   mixpanel: mixpanelReducer,
-  ingress: ingressScreenReducer
+  ingress: ingressScreenReducer,
+  landingBanners: landingScreenBannersReducer
```

Comment:
Let's add a snapshot test for the `featuresPersistor` since we edited its configuration